contracts/src/token/erc721/mod.rs

```diff
@@ -1092,7 +1088,7 @@ impl Erc721 {
                 operator,
                 from,
                 token_id,
-                data.to_vec(),
+                data.to_vec().into(),
```

Comment:
Curious if we can use just `data.into()`.

Reply:
We can't sadly, there's some weirdness here in the new types it seems :(